### DISCUSSION TITLE
HERO10 & 12 - Video Settings Change When Streaming

### DISCUSSION THREAD

**Discussion by asylthe** [gopro/labs#874] [General]
I am running the latest Labs firmware on a HERO10 & HERO12.  I'll use the HERO12 as the example, but the same thing happens on the 10.

I have my video options set to 16:9 1080 30 L+.  After using the !GL command QR Code, 16:9 | 1080 | 60 | W appears on the screen briefly when the camera goes live.  It does in fact switch to W from L or L+ (depending on what I have set) which isn't optimal for my use.

My preference would be to stream with 16:9 1080 30 L+ (or L) as my video mode.  Is there some way that I can use those settings for streaming with Labs firmware installed, or does streaming only support 16:9 1080 60 W? Not sure why it switches, or if there's a fix.  Hopefully I'm not overlooking something simple.

**Comment by dnewman-gpsw** [comment]
Livestream has a hidden preset (from my understanding), so you can't set Linear or Linear+Horizon, or EV etc.  There is no fix, it by design (I not much of a fan of these limitations either.) Livestream media is always at 30, have the local recording is at 60.

**Comment by asylthe** [comment]
I greatly appreciate the insight.  It's unfortunate those settings aren't selectable for streaming, but it will have to do I guess.  Maybe that will change in the future.
